src/api/app/views/webui2/webui/package/_submit_request_dialog.html.haml

```diff
@@ -1,55 +1,57 @@
-            = text_field_tag(:sourceproject, project.name, disabled: true, class: 'form-control text-truncate')
-          .form-group
-            = label_tag(:targetproject, 'To target project:')
-            = text_field_tag(:targetproject, target_project, disabled: params[:readonly], required: true, class: 'form-control',
-                             data: { autocomplete_url: autocomplete_projects_path,
-                                     requests_url: request_list_small_path,
-                                     develpackage_url: package_devel_project_path })
-            - if params[:readonly]
-              = hidden_field_tag(:targetproject, target_project)
-
-          .form-group
-            = label_tag(:targetpackage, 'To target package:')
-            = text_field_tag(:targetpackage, target_package, disabled: params[:readonly], class: 'form-control')
-            - if params[:readonly]
-              = hidden_field_tag(:targetpackage, target_package)
-
-          .form-group
-            = label_tag(:description, 'Description:')
-            ~ text_area_tag(:description, description, size: '40x3', class: 'form-control', required: true)
-
-          .form-group.d-none#supersede-display
-            = label_tag(:supersede_requests, 'Supersede requests:')
-            #supersede-requests
-
-          .custom-control.custom-checkbox#sourceupdate-display
-            = check_box_tag(:sourceupdate, 'cleanup', cleanup_source, class: 'custom-control-input')
-            = label_tag(:sourceupdate, 'Remove local package if request is accepted', class: 'custom-control-label')
-
-          %p.d-none#devel-project-warning
-            You are about to bypass the devel project, please submit to
-            %b#devel-project-name
-            instead.
-
-      .modal-footer
-        = render partial: 'webui2/shared/dialog_action_buttons', locals: { submit_tag_text: 'Create' }
-
-:javascript
+.modal.fade#submit-request-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'submit-request-modal-label', hidden: true } }
+  .modal-dialog.modal-dialog-centered{ role: 'document' }
+    .modal-content
+      = form_tag({ controller: 'package', action: 'submit_request' }, method: 'post') do
+        .modal-header
+          %h5.modal-title#submit-request-modal-label Create Submit Request
+        .modal-body
+          %p
+            Do you want to submit #{package_link package, rev: revision}?
+          %p.font-italic
+            You are about to send a request to the owner of the original #{package ? 'package' : 'project'}. Please describe what you have done and
+            why they should apply your changes. The owner will be notified with the following details.
+
+            = hidden_field_tag(:project, project)
+            = hidden_field_tag(:package, package)
+            = hidden_field_tag(:rev, revision)
+            .form-group
+              = label_tag(:sourceproject, 'From source project:')
```

Comment:
[nitpick] I know that you are not introducing it, but the html selectors shouldnt be snake case? i.e not `sourceproject` but `source_project` , not `targetproject`, `target_project`, not `targetpackage`, but `target_package`, would you mind to standardize it if you think that's right?

Reply:
This is a good point, but I would have to also change the bento view and the controller's action for this. I would prefer to not do it in this PR.

Fine with you?

Reply:
Ok

Reply:
Alright, since this was the one point to request changes, I will dismiss your review now.